src/components/Footer.jsx

```diff
@@ -1,7 +1,40 @@
 import React from "react";
+import {
+  FaInstagram,
+  FaEnvelope,
+  FaSquareFacebook,
+  FaLinkedin,
+} from "react-icons/fa6";
+import Image from "next/image";
+import safLogo from "../app/saf-logo.webp";
```

Comment:
replace with `import redLogo from "../../public/svg/red-logo.svg";`
All assets should come from the public folder.